pegasus/sites.v3/code.org/public/educate/csp/unit1.moved

```diff
@@ -1 +1 @@
-https://curriculum.code.org/csp/unit1
```

Comment:
Any reason not to make these relative paths?

Reply:
The relative path sends you [here](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites/virtual/curriculum-csp/index.haml), which is weirdly in the `pegasus/sites` directory, not `sites.v3` where everything else is. I didn't want to chance that going live so I used the hardcoded url.

Reply:
I don't think it's worth figuring this out at this point, but if you/others disagree I can make a ticket to follow up!

Reply:
Interesting, good catch! Yeah agreed, no objection to leaving it a mystery for now :)